Simplified/MyBooks/RSAManager.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  RSAManager.swift
+//  SimplyE
+//
+//  Created by Raman Singh on 2021-03-30.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+class RSAManager {
```

Comment:
it's best to avoid "Manager" as a class suffix, since [it's an anti pattern](coding horror manager). "Managers" have the tendency to quickly become "do everything" types, and as they grow their semantics become fuzzy. 

In this case it seems what this class does is encrypting and decrypting using RSA system, so maybe NYPLRSACypher / NYPLRSACryptographer or something like that express its duties more in detail? And if it's specific to AxisNow probably the "Axis" prefix should be added too.